lib/marklogic.js

```diff
@@ -772,6 +776,11 @@ function initClient(client, inputParams) {
       );
     }
     connectionParams.auth = 'SAML token='+token;
+  } else if(authType === 'CLOUD') {
+    if(!connectionParams.apiKey) {
+      throw new Error('apiKey needed for ML cloud authentication.');
```

Comment:
Spell out "MarkLogic" here instead of "ML" (this is something we're trying to do across all of our tools to avoid any confusion over the frequent usage of "ML" as an abbreviation for "machine learning").